apps/extension/src/Setup/Ledger/LedgerConfirmation.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { ThemeContext } from "styled-components";
+
+import {
+  Button,
+  ButtonVariant,
+  Icon,
+  IconName,
+  IconSize,
+} from "@anoma/components";
+import { shortenAddress } from "@anoma/utils";
+import { useSanitizedParams } from "@anoma/hooks";
+
+import { useRequester } from "hooks/useRequester";
+import { LedgerError } from "./Ledger.components";
+import {
+  TopSection,
+  TopSectionHeaderContainer,
+  TopSectionButtonContainer,
+  ButtonsContainer,
+  SubViewContainer,
+  UpperContentContainer,
+  Header1,
+  BodyText,
+} from "Setup/Setup.components";
+import { Ports } from "router";
+import { AddLedgerAccountMsg } from "background/ledger/messages";
+import { closeCurrentTab } from "utils";
+
+const LedgerConfirmation: React.FC = () => {
+  const navigate = useNavigate();
+  const requester = useRequester();
+  const themeContext = useContext(ThemeContext);
+  const { alias = "", address = "", publicKey = "" } = useSanitizedParams();
+  const [error, setError] = useState<string>();
+
+  const handleSubmitClick = async (): Promise<void> => {
```

Comment:
I think we can use `useCallback` here as this does not use any state.

Reply:
Done!